core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -197,6 +197,8 @@ object RequestChannel extends Logging {
           .append(",securityProtocol:").append(context.securityProtocol)
           .append(",principal:").append(session.principal)
           .append(",listener:").append(context.listenerName.value)
+          .append(",clientSoftwareName:").append(context.clientInformation.softwareName)
```

Comment:
perhaps we could just have `channelMetadata` and then `context.channelMetadata` (and have a `toString` for that class)...

Reply:
I have changed to dump the client information object. It looks like this `clientInformation:ClientInformation(softwareName=apache-kafka-java, softwareVersion=2.5.0-SNAPSHOT)` in the request log.